### ISSUE TITLE
building java client fails due to javadocs

### ISSUE THREAD

**Issue by karussell** [boldtrn/graphhopper-routing-api-swagger#6]
```txt
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-javadoc-plugin:2.10.4:jar (attach-javadocs) on project directions-api-client: MavenReportException: Error while generating Javadoc:
[ERROR] Exit code: 1 - /install/graphhopper-routing-api-swagger/java/src/main/java/com/graphhopper/directions/api/client/model/GHMatrixRequest.java:72: error: bad use of '>'
[ERROR] * The starting points for the routes. E.g. if you want to calculate the three routes A->1, A->2, A->3 then you have one from_point parameter and three to_point parameters. Is a string with the format longitude,latitude.
[ERROR] ^
[ERROR] /install/graphhopper-routing-api-swagger/java/src/main/java/com/graphhopper/directions/api/client/model/GHMatrixRequest.java:72: error: bad use of '>'
[ERROR] * The starting points for the routes. E.g. if you want to calculate the three routes A->1, A->2, A->3 then you have one from_point parameter and three to_point parameters. Is a string with the format longitude,latitude.
[ERROR] ^
[ERROR] /install/graphhopper-routing-api-swagger/java/src/main/java/com/graphhopper/directions/api/client/model/GHMatrixRequest.java:72: error: bad use of '>'
[ERROR] * The starting points for the routes. E.g. if you want to calculate the three routes A->1, A->2, A->3 then you have one from_point parameter and three to_point parameters. Is a string with the format longitude,latitude.
[ERROR] ^
[ERROR] /install/graphhopper-routing-api-swagger/java/src/main/java/com/graphhopper/directions/api/client/model/GHMatrixRequest.java:113: error: semicolon missing
[ERROR] * pecifies which arrays should be included in the response. Specify one or more of the following options 'weights', 'times', 'distances'. To specify more than one array use e.g. out_array=times&out_array=distances. The units of the entries of distances are meters, of times are seconds and of weights is arbitrary and it can differ for different vehicles or versions of this API.
[ERROR] ^
[ERROR] /install/graphhopper-routing-api-swagger/java/src/main/java/com/graphhopper/directions/api/client/model/GHResponseInstruction.java:210: error: malformed HTML
[ERROR] * optional - Only available for USE_ROUNDABOUT instructions. The radian of the route within the roundabout - 0<r<2*PI for clockwise and -2PI<r<0 for counterclockwise transit. Null if the direction of rotation is undefined.
[ERROR] ^
[ERROR] /install/graphhopper-routing-api-swagger/java/src/main/java/com/graphhopper/directions/api/client/model/GHResponseInstruction.java:210: error: malformed HTML
[ERROR] * optional - Only available for USE_ROUNDABOUT instructions. The radian of the route within the roundabout - 0<r<2*PI for clockwise and -2PI<r<0 for counterclockwise transit. Null if the direction of rotation is undefined.
[ERROR] ^
[ERROR] /install/graphhopper-routing-api-swagger/java/src/main/java/com/graphhopper/directions/api/client/model/GHResponseInstruction.java:210: error: malformed HTML
[ERROR] * optional - Only available for USE_ROUNDABOUT instructions. The radian of the route within the roundabout - 0<r<2*PI for clockwise and -2PI<r<0 for counterclockwise transit. Null if the direction of rotation is undefined.
[ERROR] ^
[ERROR] /install/graphhopper-routing-api-swagger/java/src/main/java/com/graphhopper/directions/api/client/model/GHResponseInstruction.java:210: error: malformed HTML
[ERROR] * optional - Only available for USE_ROUNDABOUT instructions. The radian of the route within the roundabout - 0<r<2*PI for clockwise and -2PI<r<0 for counterclockwise transit. Null if the direction of rotation is undefined.
[ERROR] ^

```

**Comment by boldtrn** [comment]
Oh, that's bad. I haven't tried to build it with maven, but only ran the tests from intellij, which works. Also mvn clean test works. Thanks for reporting. The issue seems to be known: https://github.com/swagger-api/swagger-codegen/issues/4753

Should we wait for a fix (issue was reported yesterday) or should we just change the swagger file?

**Comment by karussell** [comment]
Ok, sure. Lets wait ...

**Comment by boldtrn** [comment]
I could create a PR for the swagger-codegen next week, if there hasn't been a fix until then?

**Comment by boldtrn** [comment]
I fixed the Javadoc build. There were a couple of things, so I needed to change <,>,& to their code representations.

This issue should be fixed due to 9b8048577d53d5ce480199fb01357cba4dbbfac7.
